README.md

```diff
@@ -11,7 +11,7 @@ is compiled into static web pages using [Hugo].
 Local setup
 -----------
 
- 1. Install [Hugo] 0.88.1.
+ 1. Install [Hugo] 0.123.0
```

Comment:
```suggestion
 1. Install [Hugo] 0.123.8
```

It seems to use `0.123.8`

https://github.com/planetarium/snack.planetarium.dev/blob/34e6b029672436af5ea461c34f13b23fc0288a75/.github/workflows/push.yml#L12